build-logic/src/main/kotlin/shared.gradle.kts

```diff
@@ -29,12 +26,3 @@ tasks.withType(KotlinCompile::class.java) {
     languageVersion.set(KotlinVersion.KOTLIN_1_8)
   }
 }
-
-configurations.all {
-  // Pin the kotlin version
-  resolutionStrategy {
-    force(libs.kotlin.stdlib)
-    force(libs.kotlin.stdlib.jdk8)
-    force(libs.kotlin.reflect)
-  }
-}
```

Comment:
This should be unused after #936.

Reply:
Required by https://scans.gradle.com/s/zfwml5btzclo4/tests/overview?outcome=FAILED.